src/bin/dual-iir.rs

```diff
@@ -265,7 +266,7 @@ mod app {
         let shared = Shared {
             usb: stabilizer.usb,
             network,
-            settings: settings.dual_iir,
+            active_settings: settings.dual_iir,
```

Comment:
I suspect `Copy` for these structs might be confusing and hiding things. Let's remove it and do explicit `clone()`.